src/html.js

```diff
@@ -12,6 +15,12 @@ export default function HTML(props) {
           content="width=device-width, initial-scale=1, shrink-to-fit=no"
         />
         <script type="text/javascript" src="https://addevent.com/libs/atc/1.6.1/atc.min.js" async defer></script>
+        <script
+          dangerouslySetInnerHTML={{ __html: mixpnljs }}
```

Comment:
@rootedbox this is not mixpanel. It's Matomo. We want to make sure we didn't exclude Mixpanel anywhere on the site. That's really a QA check.